### DISCUSSION TITLE
how to run the example standalone?

### DISCUSSION THREAD

**Discussion by lemuellew** [ExpediaGroup/graphql-kotlin#1340] [Q&A]
I copied the `example` directory to another directory, but the operation failed. The example seems to depend on the configuration of the upper-level directory. How can I run the example standalone?

![image](https://user-images.githubusercontent.com/59448437/148638756-0c81fa1a-fb5d-4fd5-a344-d749148c0ef5.png)

after add `version "5.3.1"` 
![image](https://user-images.githubusercontent.com/59448437/148638743-9fff81fb-4a0e-4725-ba07-f58fd448c516.png)

**Comment by smyrick** [comment]
You shouldn't need to copy them to another directory. If you build the library locally, then go to the examples and run them, they should start up

**Reply by lemuellew** [reply]
Yes, I do know that the example server can be successfully run without changing the example directory, but I just want to know how to run the example independently without relying on the upper gradle configuration.

**Reply by lemuellew** [reply]
In other words, I want to use libraries and plugins from maven central instead of local in example

**Reply by dariuszkuc** [reply]
Have you seen the [original blog post](https://medium.com/expedia-group-tech/creating-a-reactive-graphql-server-with-spring-boot-and-kotlin-54aca7316470) or the [KotlinConf presentation](https://www.youtube.com/watch?v=7YJyPXjLdug&index=25) or the [docs](https://opensource.expediagroup.com/graphql-kotlin/docs/server/spring-server/spring-overview)?
